utils.py

```diff
@@ -165,7 +176,10 @@ def evaluate_prompt_format(
         structured_prompt_format, original_to_current_multiple_choice_classes, interval_ids_to_test, args.n_shot)
 
     # 2. update the output values if needed, i.e. if the multiple choice classes now have different names
-    assert all(len(dataset[idx]['output']) == 1 for idx in selected_dataset_ids)
+    assert all(len(dataset[idx]['output']) >= 1 for idx in selected_dataset_ids)
```

Comment:
Be a bit lenient here. This doesn't hold for some generation tasks. When that happens, only take the first one.

Reply:
Wouldn't it be better to raise some kind of warning in the > 1 case? Otherwise the code will silently take the first output and you might never know it happened

Reply:
Yeah makes sense. Added a warning below.